prompt.go

```diff
@@ -49,6 +60,49 @@ func (i *InitActionCmd) setParams(dirs []string) error {
 			}
 		}
 	}
+	if i.ConfigPath == "" && len(configs) > 0 {
+		prompt := promptui.Select{
+			Label: "Choose atlas.hcl config file path to use",
```

Comment:
I think there should be a way to skip `atlas.hcl`. Something like,

```
Use `atlas.hcl` file as a config?
  - No
  - Use with env "dev".
  - Use with env "local".
  - ...
```

If you have multiple matches (which rarely happens), you can do this:

```
Use `atlas.hcl` file as a config?
  - No
  - Use path/to/atlas.hcl.
  - Use path/to/atlas.hcl.

Choose an env:
  - ...
  - ...
```

Reply:
Also, please use HideHelp in prompui - https://github.com/ariga/atlas/blob/master/cmd/atlas/internal/cmdapi/schema.go#L744

This will hide the annoying navigation message.